README.md

```diff
@@ -48,29 +48,13 @@ $ cargo add async-mongodb-session
 
 ## Configuration
 
-This library utilises the document [expiry feature](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-after-a-specified-number-of-seconds) in mongodb to expire the session.
+This library utilises the document [expiry feature](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-at-a-specific-clock-time) in mongodb to expire the session at the expiry.
 
 The management of the expiry feature fits into the 12 factor [admin process definintion](https://12factor.net/admin-processes) so it's recommended to use an process outside of your web application to manage the expiry parameters.
 
-A `created` property is available on the root of the session document that so the [expiry feature](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-after-a-specified-number-of-seconds) can be used in the configuration.
+A `expireAt` property is available on the root of the session document IFF the session expire is set. Note that  [async-session doesn't set by default](https://github.com/http-rs/async-session/blob/main/src/session.rs#L98).
 
-If your application code to create a session store is something like:
-```
-let store = MongodbSessionStore::connect("mongodb://127.0.0.1:27017", "db_name", "coll_session").await?;
-```
-
-Then the script to create the expiry would be:
-```
-use db_name;
-db.coll_session.createIndex( { created": 1 } , { expireAfterSeconds: 300 } ) 
-```
-
-If you wish to redefine the session duration then the index must be dropped first using:
-```
-use db_name;
-db.coll_session.dropIndex( { "created": 1 })
-db.coll_session.createIndex( { created": 1 } , { expireAfterSeconds: 300 } ) 
-```
+To enable this [expiry feature](https://docs.mongodb.com/manual/tutorial/expire-data/#expire-documents-at-a-specific-clock-time) at `index` for `expireAt` is created when the client `connet`. Allowing then to use this property from the session to set the expiration of the document.
 
```

Comment:
I think we should keep both approaches in the README as well as state when you would use one over the other.